index.html

```diff
@@ -109,6 +117,8 @@ <h3>Paste it into your Github issue</h3>
         output.value += "\n* __Request:__ " + report.request;
         output.value += "\n* __Content Language:__ " + report.content_language;
         output.value += "\n* __Service:__ " + report.service;
+        output.value += "\n* __GMT time:__" + report.time;
+        output.value += "\n* __Package:__" + report.package;
```

Comment:
We did not include the package by purpose. It should always be org.schabi.newpipe when people post the report at our GitHub page. The field exists for our Sentry importer to filter out reports from forks and copies. Please remove this line again.

I am also not sure about the time step. Why do we need it? Posting the exact timestamp harms a users privacy and leaks even more personal information in the crash report.

Reply:
Ok, I understand.

About timestamp: then why you collect exact timestamp and send it by email? Why not only day, or at least just hour?

Reply:
Good question. I think this was initially added to prevent spam. But I am not sure. I'll check this later.